### PR TITLE
set other solvers to be solved

### PR DESCRIPTION
**Pull request by harshad16**
set other solvers to be solved
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: core/overlays/stage/common/configmaps.yaml [hunk 1]
```diff
@@ -37,6 +37,10 @@ metadata:
 data:
   solvers: |
     solver-rhel-8-py36
+    solver-fedora-31-py38
+    solver-fedora-31-py37
```
Comment (fridex) [inline core/overlays/stage/common/configmaps.yaml:41]: Do we want to invest time to Fedora 31? I think we could include F33 instead.

Comment (fridex) [reply core/overlays/stage/common/configmaps.yaml:41]: https://endoflife.software/operating-systems/linux/fedora

Comment (harshad16) [reply core/overlays/stage/common/configmaps.yaml:41]: ack, will remove the f31.

**Review by fridex — APPROVED** [review]
:100:

**Pull request merged**